From 20a921ede113784c3b2acd7ee2f884813a0433ea Mon Sep 17 00:00:00 2001
From: Alex Porosanu <alexandru.porosanu@freescale.com>
Date: Fri, 17 Jul 2015 15:03:48 +0300
Subject: [PATCH 320/451] crypto/caam: ensure relevant accessors are available

For ARM in BE mode, some acessors are not available. This
patch adds the necessary glue code so that the codebase
remains the same

Change-Id: I8c9ee9ba396306a6abc837886615d2103d9334f9
Signed-off-by: Alex Porosanu <alexandru.porosanu@freescale.com>
[Xulin: Original patch taken from
Linux-LS1043A-SDK-V0.4-SOURCE-20150826-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/crypto/caam/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/crypto/caam/Kconfig b/drivers/crypto/caam/Kconfig
index c0ceff8..4ec6ec4 100644
--- a/drivers/crypto/caam/Kconfig
+++ b/drivers/crypto/caam/Kconfig
@@ -158,7 +158,7 @@ config CRYPTO_DEV_FSL_CAAM_JR_UIO
 
 config CRYPTO_DEV_FSL_CAAM_DIFFERENT_ENDIAN
 	bool "Mixed endianness system"
-	depends on CRYPTO_DEV_FSL_CAAM
+	depends on CRYPTO_DEV_FSL_CAAM && !CPU_BIG_ENDIAN
 	default n
 	help
 	  There are SoCs in which the core's endianness is different from the
-- 
1.7.5.4

